changes/2020-07-14_multi-keyring-require-generation/change.md

```diff
@@ -0,0 +1,108 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Require Generation in the Multi-Keyring
+
+## Affected Features
+
+This serves as a reference of all features that this change affects.
+
+| Feature                                                                                           |
+| ------------------------------------------------------------------------------------------------- |
+| [Multi-Keyring](../../framework/multi-keyring.md)                                                 |
```

Comment:
Quick extra note. I noticed in another PR you had the specific SHA the feature it changed as opposed to just the link. You should probably use the SHA here and everywhere else multi-keyring.md is linked

Reply:
Agreed, although whether the following section needs it as well is still up in the air: https://github.com/awslabs/aws-encryption-sdk-specification/pull/154#discussion_r455213501